docs/Project-Setup.md

```diff
@@ -0,0 +1,43 @@
+## Setup local repository
+
+Clone the repo.
+
+```sh
+git clone https://github.com/ERNI-Academy/starterkit-mobile-application-flutter.git
+```
+
+Untrack the files under the folder `.secrets` by adding it in your `.gitignore` file.
+
+```sh
+.secrets/**
```

Comment:
I'd suggest to add that to this repo's gitignore file. As the (default) secrets are already checked in, they won't get lost.
And if we maintainers see that the defaults need an update, we can always force add the changes.